core/Cargo.toml

```diff
@@ -75,6 +75,8 @@ tokio-codec = "0.1"
 tokio-fs = "0.1"
 tokio-io = "0.1"
 untrusted = "0.7.0"
+lazy_static = "1.4.0"
```

Comment:
I don't see where lazy_static is now used in `core/`?

Reply:
Oh yeah this is unnecessary.